src/overlays/actors/ovl_En_Mnk/z_en_mnk.c

```diff
@@ -15,8 +16,34 @@ void EnMnk_Destroy(Actor* thisx, PlayState* play);
 void EnMnk_Update(Actor* thisx, PlayState* play);
 void EnMnk_Draw(Actor* thisx, PlayState* play);
 
-#if 0
-ActorInit En_Mnk_InitVars = {
+// Draw funcs
+void EnMnk_TiedUpMonkey_Draw(Actor* thisx, PlayState* play);
+void EnMnk_HangingMonkey_Draw(Actor* thisx, PlayState* play);
+
+// Action funcs
+void EnMnk_Monkey_WaitToRunAndWaitAtEachPoint(EnMnk* this, PlayState* play);
+void EnMnk_TiedUpMonkey_Wait(EnMnk* this, PlayState* play);
+void EnMnk_HangingMonkey_StruggleBeforeDunk(EnMnk* this, PlayState* play);
+void EnMnk_Monkey_WaitToTalkAfterRun(EnMnk* this, PlayState* play);
+void EnMnk_Monkey_WaitForPlayerApproach(EnMnk* this, PlayState* play);
+void EnMnk_Monkey_WaitToFollowPath(EnMnk* this, PlayState* play);
+void EnMnk_TiedUpMonkey_WaitUnused(EnMnk* this, PlayState* play);
+void EnMnk_TiedUpMonkey_WaitForInstrument(EnMnk* this, PlayState* play);
+void EnMnk_TiedUpMonkey_WaitForCutRope(EnMnk* this, PlayState* play);
+void EnMnk_HangingMonkey_WaitAfterDunk(EnMnk* this, PlayState* play);
+void EnMnk_Monkey_WaitToTalkAfterSaved(EnMnk* this, PlayState* play);
```

Comment:
Just an open question for everyone. I wonder if it's worth doing:

```suggestion
void EnMnk_Monkey_WaitToFollowPath(EnMnk* this, PlayState* play);
void EnMnk_MonkeyTiedUp_WaitUnused(EnMnk* this, PlayState* play);
void EnMnk_MonkeyTiedUp_WaitForInstrument(EnMnk* this, PlayState* play);
void EnMnk_MonkeyTiedUp_WaitForCutRope(EnMnk* this, PlayState* play);
void EnMnk_MonkeyHanging_WaitAfterDunk(EnMnk* this, PlayState* play);
void EnMnk_Monkey_WaitToTalkAfterSaved(EnMnk* this, PlayState* play);
```
to have `Monkey` as a common base. Similarly:
```c
typedef enum {
    /* 0 */ TIEDUPMONKEY_ANIM_KICKAROUND,
    /* 1 */ TIEDUPMONKEY_ANIM_KICKUPANDDOWN,
    /* 2 */ TIEDUPMONKEY_ANIM_SHH,
```
vs.
```c
typedef enum {
    /* 0 */ MONKEYTIEDUP_ANIM_KICKAROUND,
    /* 1 */ MONKEYTIEDUP_ANIM_KICKUPANDDOWN,
    /* 2 */ MONKEYTIEDUP_ANIM_SHH,
```

Reply:
I say yes, but also not too big a deal atm.

Reply:
Here's what it would look like:
https://github.com/gamestabled/mm/compare/en_mnk...engineer124:mm:en_mnk?expand=1
Could quickly PR & merge again if we're fine with it